README.md

```diff
@@ -11,10 +11,36 @@ Common client code - in go - for ONS APIs:
 * healthcheck -> health
 * hierarchy
 * identity
+* image
 * importapi
 * renderer
 * search
 
+
+#### Usage
+
+(WIP) Each client defines two constructor functions: one that creates a new healthcheck client (with a new dp-net/http Clienter), and the other that allows you to provide it externally, so that you can reuse it among different clients.
+
+For example, you may create a new image API client like so:
+```
+    import  "github.com/ONSdigital/dp-api-clients-go/image"
+
+    ...
+    imageClient := image.NewAPIClient(<url>)
+    ...
+```
+
+Or you may create it providing a Healthcheck client:
+```
+    import  "github.com/ONSdigital/dp-api-clients-go/image"
+    import  "github.com/ONSdigital/dp-api-clients-go/health"
+
+    ...
+    hcClient := health.NewClient(<genericName>, <url>)
+    imageClient := image.NewWithHealthClient(hcClient)
```

Comment:
We had a discussion with Eleonor about this: https://trello.com/c/R9Yl8Axr/4804-investigate-whether-we-can-consolidate-api-config-values-without-merging-clients-2d

 Originally, I created something like this:
 ```NewClientWithClienter(name, url string, clienter dphttp.Clienter) *Client```
You're right about maybe not being very clear. After the discussion about the URL (which would need to be provided in both cases), I did the change to pass the Health Client to the constructor instead of the clienter (so the API would be a kind of 'extension' of healthcheck client). My initial intention was to pass the clienter.